variod.c

```diff
@@ -136,26 +138,72 @@ int create_xcsoar_connection()
 
 }
 
+int multi_sentence_clean(char *msg,int len)
+{
```

Comment:
What is the intention of this function? What "clean" means here? What does it return? The outline of the algorythm would be helpful in a comment.